src/content/roadmap/_index.en.md

```diff
@@ -5,15 +5,24 @@ weight = 40
 pre = "<b>8. </b>"
 +++
 
-This is a preliminary community roadmap. It represents our best guess at broad priorities, and can serve as a general guideline.
+Submariner organizes all current and upcoming work using GitHub Issues, Projects, and Milestones.
 
-Details of past releases are described [here](../releases).
+## Planning Process
 
-## v0.7.0
+In preparation for sprint planning meetings (see [SubM's GCal][cal]), GitHub Issues should be raised for work that is to be a part of a
+sprint.  Issues targeted for a sprint should be added to the upcoming Project's "Backlog" column. During sprint planning meetings, Issues
+will be discussed and moved to the "TODO" column and the milestone will be set to the targeted release. As contributors make progress during
```

Comment:
```suggestion
will be discussed and moved to the "TO DO" column and the milestone will be set to the targeted release. As contributors make progress during
```

Reply:
The column is "TO DO", but I think it should be "TODO". As I understand, it should either be one word or hyphenated. In code it's typically one word.

https://english.stackexchange.com/questions/46217/todo-list-or-to-do-list